analyzers/tests/SonarAnalyzer.UnitTest/SymbolicExecution/Roslyn/RoslynSymbolicExecutionTest.Loops.cs

```diff
@@ -182,5 +182,67 @@ public void Loops_FalseConditionNotExecuted(string loop)
 Tag(""End"");";
             SETestContext.CreateCS(code).Validator.ValidateTagOrder("End");
         }
+
+        [TestMethod]
+        public void ForLoopWithTryCatchAndNullFlows()
```

Comment:
This change has nothing to do with loops. Loop was just a reproducer scenario.

Let's delete this and move this test case to the NRE rule test cases.